cmd/mapt/cmd/aws/services/openshift-snc.go

```diff
@@ -72,9 +72,10 @@ func createSNC() *cobra.Command {
 					Version:        viper.GetString(ocpVersion),
 					Arch:           viper.GetString(params.LinuxArch),
 					PullSecretFile: viper.GetString(pullSecretFile),
-					CaCertFile:     viper.GetString(caCertFile),
-					Spot:           viper.IsSet(awsParams.Spot),
-					Timeout:        viper.GetString(params.Timeout)}); err != nil {
+					CaCertFile: util.If(viper.GetString(caCertFile) == "",
```

Comment:
mmm I would suggest to remove default value for caCert param...and use `viper.IsSet` to add the value if exists as param....also in case it is not a param..you can leave it as nil...and the handle the creation inside of the action...not rom here...as all logic should be at action level